File: contains_duplicate/contains_duplicate.go
package main

func containsDuplicate(nums []int) bool {
    nums_len := len(nums)
    for currindex, currvalue := range nums {
        for i := currindex+1; i < nums_len; i++ {
            if nums[i] == currvalue {
                return true
            }
        }
    }
    return false
}

File: arrays/contains_duplicate/contains_duplicate.c
bool containsDuplicate(int* nums, int numsSize) {
    int i;
    int j;
    for (i = 0; i < numsSize; i++) {
        int currentIndex = i;
        int currentValue = nums[i];
        for (j = i+1; j < numsSize; j++) {
            if (nums[j] == currentValue) {
                return true;
            }
        }
    }
    return false; 
}
